src/edu/cscc/Converter.java

```diff
@@ -0,0 +1,92 @@
+package edu.cscc;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * This class takes a list of employees unmarshalled from an XML file as POJOs and converts the data into
+ * appropriated JSON format.  It contains two methods, one for converting the employees and one for comverting
+ * the stores.
+ *
+ * <p>
+ * Author: Elton Pierce
+ * Date: 10/25/2023
+ */
+public class Converter {
+
+    private Employees employees;
+
+    // Constructor
+    public Converter(Employees employees) {
+        this.employees = employees;
+    }
+
+
+    /**
+     * This method converts the employees POJOs into appropriate JSON format.
+     */
+    public List<LinkedHashMap<String, Object>> convertEmployees() {
+
+        List<LinkedHashMap<String, Object>> empConverted = new ArrayList<LinkedHashMap<String, Object>>();
+
+        for (int i = 0; i < employees.getEmployees().size(); i++) {
+
+            empConverted.add(new LinkedHashMap<>());
+
+            empConverted.get(i).put("firstName", employees.getEmployees().get(i).getFirstName());
+            empConverted.get(i).put("lastName", employees.getEmployees().get(i).getLastName());
+            empConverted.get(i).put("payRate", employees.getEmployees().get(i).getPayRate());
+            empConverted.get(i).put("id", employees.getEmployees().get(i).getId().substring(0, employees.getEmployees().get(i).getId().length() - 1));
+
+            if (employees.getEmployees().get(i).getId().endsWith("A")) {
+                empConverted.get(i).put("role", "associate");
+            } else if ((employees.getEmployees().get(i).getId().endsWith("S"))) {
+                empConverted.get(i).put("role", "shift_manager");
+            } else {
+                empConverted.get(i).put("role", "store_manager");
+            }
+
+            empConverted.get(i).put("storeNumber", employees.getEmployees().get(i).getActiveStore().getStoreNumber());
+
+        }
+
+        return empConverted;
+    }
```

Comment:
This isn an interesting approach. Consider that rather than a HashMap<<String, Object> to represent the properties of a valid employee, we could use another class to represent them instead.